CSS/index.css

```diff
@@ -35,6 +35,84 @@
   margin: 0;
   font-size: 48px;
 }
+/*Import Menu.less file
+*/
+.menu-button {
+  position: absolute;
+  left: 20px;
+  cursor: pointer;
+}
+.menu {
+  display: block;
+  max-width: 0px;
+  overflow: hidden;
+  transition: all 0.25s ease-in-out;
+  position: fixed;
+  top: 75px;
+  left: 0;
+  width: 350px;
+  height: 100vh;
+  /* This give the height 100% of the screen or, '100% of View Height' */
+  background-color: #81C784;
+  border: 1px solid #004D40;
+  border-left: none;
+  border-bottom: none;
+  z-index: 2;
+  /* This set the layer level of the element, it has precendence over initial level of 1*/
+}
+.menu ul li {
+  list-style-type: none;
+  font-size: 32px;
+  padding: 30px 0;
+  text-align: center;
+}
+.menu--open {
+  max-width: 351px;
+  transition: all 0.25s ease-in-out;
+}
```

Comment:
I would use ```transform: translateX()``` here to prevent the text being squashed as the menu opens and closes